deploy/package.sh

```diff
@@ -23,8 +24,9 @@ SYNAPSE_POOL=${12:-${SYNAPSE_POOL}}
 
 DEPLOY_PGSQL=${13:-${DEPLOY_PGSQL:-"true"}}
 
-set -ex
+MODE="batch-account$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres")"
```

Comment:
MODE="batch-account$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres" || echo '')"

Reply:
note that I added the following at the end to prevent stopping after failure if DEPLOY_PGSQL == "true".

| echo ''